### ISSUE TITLE
Bug in predictPlp function

### ISSUE THREAD

**Issue by nbehzad** [OHDSI/PatientLevelPrediction#314]
**Describe the bug**
`plpModel$preprocessing` is used in the if statements of the `predictPlp` function. However, the `plpModel` includes `preprocess`. As a result, some of the if statements, such as the one below, will never be TRUE:
```sh
  if(!is.null(plpModel$preprocessing$featureEngineering)){
    plpData <- do.call(
      applyFeatureengineering, 
      list(
        plpData = plpData,
        settings = plpModel$preprocessing$featureEngineering
      )
    )
    featureEngineering <- T
  } else{
    featureEngineering <- F
  }
```

**Comment by egillax** [comment]
This was supposed to have been fixed in #310. All preprocess should now be preprocessing. But I see that there's still some use of preprocess in the code. So I:

- Changed preprocess to preprocessing everywhere (saving and loading, feature importance and sklearnClassifier)
- If loading an older model that saved this object as preprocess, still load it correctly into an object called preprocessing so the rest of the code will run correctly

Should be fixed now in develop.

**Comment by jreps** [comment]
fixed in the main branch now
